components/tyk-dashboard/templates/deployment-dashboard.yaml

```diff
@@ -0,0 +1,255 @@
+{{- if .Values.enabled -}}
```

Comment:
any possibilities for disabling dashboard deployment while using the dashboard chart? If users use this chart, it should be enabled in my opinion. what do you think?

Reply:
Deleted as it was indeed useless when controlled at component level.